applications/FluidDynamicsApplication/tests/test_navier_stokes_compressible_explicit_solver.py

```diff
@@ -24,20 +24,88 @@ def test_DimensionNumber(self):
         
```

Comment:
I think this shouldn't be tested through the solver but in the `test_fluid_characteristic_numbers_utilities.cpp`.

Reply:
Agreed.

Reply:
Agreed